app/scripts/services/secrets.js

```diff
@@ -28,17 +28,29 @@ angular.module("openshiftConsole")
       return secretsByType;
     };
 
+    var setDataParams = function(decodedSecretData, serverName, data) {
```

Comment:
If this is specific to dockercfg, we should have that somewhere in the function name.

Reply:
This method is not intended to be specific for `dockercfg` or `config.json`. It's purpose is to assemble an object that will contain server address, username, password.